activerecord/lib/active_record/autosave_association.rb

```diff
@@ -22,7 +22,7 @@ module ActiveRecord
   #
   # == Validation
   #
-  # Children records are validated unless <tt>:validate</tt> is +false+.
+  # Child records are validated unless <tt>:validate</tt> is +false+.
```

Comment:
@vipulnsward actually, @meinac's change is correct - "child records" is a compound noun so only the last noun is plural. However this is English so there are exceptions to that rule but this isn't one of them:

https://en.wikipedia.org/wiki/English_plurals#Plurals_of_compound_nouns